doc/changes.rst

```diff
@@ -0,0 +1,95 @@
+
+* Let surface return xr.DataArray instead of xr.Dataset (`#408 <https://github.com/GenericMappingTools/pygmt/pull/408>`__)
+* Update GMT constant GMT_STR16 to GMT_VF_LEN for GMT API change in 6.1.0 (`#397 <https://github.com/GenericMappingTools/pygmt/pull/397>`__)
+* Properly trigger pytest matplotlib image comparison (`#352 <https://github.com/GenericMappingTools/pygmt/pull/352>`__)
+* Use uuid.uuid4 to generate unique names (`#274 <https://github.com/GenericMappingTools/pygmt/pull/274>`__)
+
+Maintenance:
+
+* Quickfix Zeit Now miniconda installer link to anaconda.com (`#413 <https://github.com/GenericMappingTools/pygmt/pull/413>`__)
+* Fix Github Pages deployment from Travis (`#410 <https://github.com/GenericMappingTools/pygmt/pull/410>`__)
+* Update and clean TravisCI configuration (`#404 <https://github.com/GenericMappingTools/pygmt/pull/404>`__)
+* Quickfix min elevation for new SRTM15+V2.1 earth relief grids (`#401 <https://github.com/GenericMappingTools/pygmt/pull/401>`__)
+* Wrap docstrings to 79 chars and check with flake8 (`#384 <https://github.com/GenericMappingTools/pygmt/pull/384>`__)
+* Update continuous integration scripts to 1.2.0 (`#355 <https://github.com/GenericMappingTools/pygmt/pull/355>`__)
+* Use Zeit Now to deploy doc builds from PRs (`#344 <https://github.com/GenericMappingTools/pygmt/pull/344>`__)
+* Move gmt from requirements.txt to CI scripts instead (`#343 <https://github.com/GenericMappingTools/pygmt/pull/343>`__)
+* Change py.test to pytest (`#338 <https://github.com/GenericMappingTools/pygmt/pull/338>`__)
+* Add Google Analytics to measure site visitors (`#314 <https://github.com/GenericMappingTools/pygmt/pull/314>`__)
+* Register mpl_image_compare marker to remove PytestUnknownMarkWarning (`#323 <https://github.com/GenericMappingTools/pygmt/pull/323>`__)
+* Disable Windows CI builds before PR `#313 <https://github.com/GenericMappingTools/pygmt/pull/313>`__ is merged (`#320 <https://github.com/GenericMappingTools/pygmt/pull/320>`__)
+* Enable Mac and Windows CI on Azure Pipelines (`#312 <https://github.com/GenericMappingTools/pygmt/pull/312>`__)
+* Fixes for using GMT 6.0.0rc1 (`#311 <https://github.com/GenericMappingTools/pygmt/pull/311>`__)
+* Assign authorship to "The PyGMT Developers" (`#284 <https://github.com/GenericMappingTools/pygmt/pull/284>`__)
+
+Deprecations:
+
+* Remove mention of gitter.im (`#405 <https://github.com/GenericMappingTools/pygmt/pull/405>`__)
+* Remove portrait (-P) from common options (`#339 <https://github.com/GenericMappingTools/pygmt/pull/339>`__)
+* Remove require.js since WorldWind was dropped (`#278 <https://github.com/GenericMappingTools/pygmt/pull/278>`__)
+* Remove Web WorldWind support (`#275 <https://github.com/GenericMappingTools/pygmt/pull/275>`__)
+
+This release contains contributions from:
+
+* Dongdong Tian
+* Wei Ji Leong
+* Leonardo Uieda
+* Liam Toney
+* Brook Tozer
+* Claudio Satriano
+* Cody Woodson
+* Mark Wieczorek
+* Philipp Loose
+* Kathryn Materna
```

Comment:
Just wondering if we should link people's name to their Github account, e.g. like how xarray does it at https://xarray.pydata.org/en/stable/whats-new.html.